docs/pages/management/guides/gcp-tags.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: GCP Labels and Tags as Teleport Node Labels
```

Comment:
"Nodes" should be "agents" since it refers to labeling SSH nodes, applications, etc.